fuse_models/include/fuse_models/unicycle_2d_state_cost_functor.h

```diff
@@ -147,28 +145,39 @@ bool Unicycle2DStateCostFunctor::operator()(
   const T* const acc_linear2,
   T* residual) const
 {
-  T position_pred[2];
-  T yaw_pred[1];
+  T delta_position_pred[2];
+  T delta_yaw_pred[1];
   T vel_linear_pred[2];
   T vel_yaw_pred[1];
   T acc_linear_pred[2];
+  T init_position[2];
+  T init_yaw[1];
+  init_position[0] = (T)0.0;
+  init_position[1] = (T)0.0;
+  init_yaw[0] = (T)0.0;
```

Comment:
I believe it's more common to see the following:
```suggestion
  init_position[0] = T(0.0);
  init_position[1] = T(0.0);
  init_yaw[0] = T(0.0);
```